hack/util.sh

```diff
@@ -641,7 +641,10 @@ function install_registry {
 }
 
 function wait_for_registry {
-	wait_for_command '[[ "$(oc get endpoints docker-registry --output-version=v1 --template="{{ if .subsets }}{{ len .subsets }}{{ else }}0{{ end }}" --config=${ADMIN_KUBECONFIG} || echo "0")" != "0" ]]' $((5*TIME_MIN))
+	local config="--config=${ADMIN_KUBECONFIG}"
+	wait_for_command "oc get endpoints docker-registry --output-version=v1 --template='{{ len .subsets }}' $config | grep -q '[1-9][0-9]*'" $((5*TIME_MIN))
+	local readyjs='{.items[*].status.conditions[?(@.type=="Ready")].status}'
+	wait_for_command "oc get pod -l deploymentconfig=docker-registry --output-version=v1 -o jsonpath='$readyjs' $config | grep -qi true" $TIME_MIN
```

Comment:
Same `$KUBECONFIG` comment as above... less important for `readyjs` as we can be sure the literal contains no whitespace for now.